clippy_lints/src/manual_div_ceil.rs

```diff
@@ -138,9 +154,30 @@ fn build_suggestion(
     applicability: &mut Applicability,
 ) {
     let dividend_sugg = Sugg::hir_with_applicability(cx, lhs, "..", applicability).maybe_par();
+    let type_suffix = if cx.typeck_results().expr_ty(lhs).is_numeric()
+        && !is_include_type_suffix(cx, lhs)
+        && matches!(lhs.kind, ExprKind::Lit(_) | ExprKind::Unary(_, _))
+    {
+        format!("_{}", cx.typeck_results().expr_ty(rhs))
+    } else {
+        String::new()
+    };
```

Comment:
If it is a numeric and a literal (e.g. 3) or unary (e.g. -3) that needs to be type-suffixed, make the type suffix to match the type of rhs here